engine-tests/src/tests/contract_call.rs

```diff
@@ -30,6 +30,14 @@ fn setup_test() -> (AuroraRunner, Signer, Address, Tester) {
     (runner, signer, token, tester)
 }
 
+#[test]
+#[should_panic]
+fn test_deploy_erc20_token_with_invalid_account_id() {
+    let mut runner = AuroraRunner::new();
+    let invalid_nep141 = "";
```

Comment:
Now it's a default and valid `AccountId`.

Reply:
Exactly, I will update it with an invalid accountId.